apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -316,14 +317,11 @@ def render(element_html: str, data: pl.QuestionData) -> str:
 
         html_params["feedback"] = partial_score.get("feedback", None)
 
-        with open(NUMBER_INPUT_MUSTACHE_TEMPLATE_NAME, "r", encoding="utf-8") as f:
-            return chevron.render(f, html_params).strip()
+        return chevron.render(template, html_params).strip()
 
     elif data["panel"] == "answer":
         ans_true = None
-        if pl.get_boolean_attrib(
-            element, "show-correct-answer", SHOW_CORRECT_ANSWER_DEFAULT
-        ):
+        if show_correct_answer:
             ans_true = format_true_ans(element, data, name)
 
         if ans_true is None:
```

Comment:
Same weird logic as fixed in most recent commit.

Reply:
Thanks, I think I caught all of them.